server/notification-providers/wecom.js

```diff
@@ -30,26 +31,50 @@ class WeCom extends NotificationProvider {
     /**
      * Generate the message to send
      * @param {object} heartbeatJSON Heartbeat details (For Up/Down only)
+     * @param {object} monitorJSON Monitor details
      * @param {string} msg General message
      * @returns {object} Message
      */
-    composeMessage(heartbeatJSON, msg) {
-        let title;
-        if (msg != null && heartbeatJSON != null && heartbeatJSON["status"] === UP) {
-            title = "UptimeKuma Monitor Up";
-        }
-        if (msg != null && heartbeatJSON != null && heartbeatJSON["status"] === DOWN) {
-            title = "UptimeKuma Monitor Down";
+    composeMessage(heartbeatJSON, monitorJSON, msg) {
+        let params = {};
+        if (heartbeatJSON != null) {
+            params = {
+                msgtype: "markdown",
+                markdown: {
+                    content: `## [${this.statusToString(
+                        heartbeatJSON["status"]
+                    )}] ${monitorJSON["name"]} \n> ${
+                        heartbeatJSON["msg"]
+                    } \n> Time (${heartbeatJSON["timezone"]}): ${
```

Comment:
Including time like this makes it quite distracting.
Does WeChat support cards or allow html-codes like `<sub>Time..</sub>` making this smaller?

Reply:
I don't see there's support for `sub`, but there are other types of messages. Like `template_card`. I think for a simple message, it's a little over complicated. Maybe use bold for msg, and normal font for timestamp?

Reply:
`template_card` seems like a better option as this looks more flexible (google translate is terrible on this text => might be missing context)

Reply:
`template_card` does look much nicer, good advice!